templates/azure/README.md

```diff
@@ -0,0 +1,27 @@
+# Azure DevOps Templates for Mender
+
+`mendersoftware/mender-ci-tools` image is used for the jobs. It's based on `debian` with pre-installed `mender-cli` and `mender-artifact` tools.
+
+Pipelines examples can be found in the related [folder](../../examples/azure/).
+
+## Templates
+The templates are [Azure DevOps step templates with parameters](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops). The step templates gives flexibility in using them in a custom stages and jobs, and allows to build pipelines to work across different environments.
+
+### Upload Artifact
+[Upload Mender Artifact](./mender-artifact-upload.yml) template uploads a Mender artifacts to a Mender server.
+
+The template's parameters:
+- `mender_uri`: Mender server URL.
+- `mender_pat`: Mender Personal Access Token (read the [documentation](https://docs.mender.io/server-integration/using-the-apis#personal-access-tokens) for more information)
+- `mender_artifact`: Relative to $(System.DefaultWorkingDirectory) path of Mender Artifact file
+
+### Create Deployment
+[Create deployment on Mender server](./mender-deployment-create.yml) template creates a deployment on a Mender server.
+
+The template's parameters:
+- `mender_uri`: Mender server URL.
+- `mender_pat`: Mender Personal Access Token (read the [documentation](https://docs.mender.io/server-integration/using-the-apis#personal-access-tokens) for more information)
+- `mender_deployment_name`: The name of Mender deployment.
+- `mender_release_name`: The name of Mender release.
+- `mender_deployment_group`: The name of Mender devices group a deployment will be triggered to. One of `mender_deployment_group` or `mender_devices_list` is required. `mender_devices_list` takes the priority if both are set.
+- `mender_devices_list`: The list of Mender devices a deployment will be triggered to.
```

Comment:
As `mender_deployment_group` is referencing `mender_devices_list`, I suggest se swap their positions:
```suggestion
- `mender_devices_list`: The list of Mender devices a deployment will be triggered to.
- `mender_deployment_group`: The name of Mender devices group a deployment will be triggered to. One of `mender_deployment_group` or `mender_devices_list` is required. `mender_devices_list` takes the priority if both are set.
```

Alternatively, leave them in the correct order but move the "One of `mender_deployment_group` or `mender_devices_list` is required. `mender_devices_list` takes the priority if both are set." to the second item.